### PR TITLE
[enh] Increase fail2ban maxretry on user login, narrow nginx log files

### PR DESCRIPTION
**Pull request by opi**
We're intended to a general public, after all ;)

### CODE REVIEW

**Comment by Psycojoker** [comment]
Is this link to #215?

**Comment by opi** [comment]
#215 only makes me have a look into fail2ban configuration ;)

**Comment by Psycojoker** [comment]
Can we display this on the user interface? After a failed login

**Comment by opi** [comment]
I don't know; From my tests, It looks like a timeout after 3 failed login. I got to investigate fail2ban configuration

File: data/templates/fail2ban/jail.conf [hunk 1]
```diff
@@ -581,4 +581,5 @@ enabled  = true
 port     = http,https
 protocol = tcp
 filter   = yunohost
-logpath  = /var/log/nginx/*.log
+logpath  = /var/log/nginx*/*error.log
```
Comment (zamentur) [inline data/templates/fail2ban/jail.conf]: I checked on 2 distinct yunohost, I have one `nginx` directory but no need `nginx*`. Are you sure it's needed ?

Comment (opi) [inline data/templates/fail2ban/jail.conf]: nope, it's probably not needed; It's only copy/pasta from others nginx sections in jail.conf (https://github.com/YunoHost/yunohost/blob/unstable/data/templates/fail2ban/jail.conf#L554-L576)

**Comment by opi** [comment]
should we replace every `nginx*` occurences with `nginx` ? I don't really care, but I want consistent configuration across the whole file !

**Comment by opi** [comment]
up!

as #215 is fixed and released (aka "fail2ban will work on user login"), I really like this improvment to be merged as well, because 2 or 3 wrong password attemp seems low to me.

**Comment by Psycojoker** [comment]
@opi would like this PR to be merged in 2.5 before the stable release. Do we have any opposition with that? I'm ok with merging it.

**Comment by alexAubin** [comment]
Yup I'm okay with it

**Comment by julienmalik** [comment]
ok let's merge this one

**Comment by Psycojoker** [comment]
That's enough positive opinions :)

**Pull request merged**